mattermost.go

```diff
@@ -29,7 +29,9 @@ import (
 	_ "github.com/mattermost/platform/model/gitlab"
 
 	// Enterprise Deps
+	_ "github.com/dgryski/dgoogauth"
 	_ "github.com/go-ldap/ldap"
+	_ "github.com/jwilander/rsc/qr"
```

Comment:
Yeah, if you can make a repo called rsc and give me write access I can move it over